keps/sig-network/2086-service-internal-traffic-policy/README.md

```diff
@@ -233,9 +242,7 @@ _This section must be completed when targeting beta graduation to a release._
 * **How can an operator determine if the feature is in use by workloads?**
 
 * Check Service to see if `internalTrafficPolicy` is set to `Local`.
-* A per-node "blackhole" metric will be added to kube-proxy which represent Services that are being intentionally dropped (internalTrafficPolicy=Local and no endpoints).
-
-TODO: add metric name once it's decided
+* A per-node "blackhole" metric will be added to kube-proxy which represent Services that are being intentionally dropped (internalTrafficPolicy=Local and no endpoints). The metric will be named `kubeproxy/sync_proxy_rules/blackhole_total` (subject to rename).
```

Comment:
Do we want to qualify this with a label for internal/external?

For names, maybe "policy_local_no_endpoints" or something.

Reply:
Good call -- renamed to `kubeproxy/sync_proxy_rules_no_endpoints_total`, and added a note to add two labels for internal/external policy values.